yoti-stub/src/services/YotiRequestProcessor.ts

```diff
@@ -80,16 +87,29 @@ export class YotiRequestProcessor {
 	 */
 	async createSession(event: APIGatewayProxyEvent, incomingPayload: any): Promise<Response> {
 		this.logger.info("START OF CREATESESSION")
+
 		const fullName = incomingPayload.resources.applicant_profile.full_name;
-		const lastUuidChars = fullName.match(/\d+/g)[0];
-		const firstTwoChars = lastUuidChars.slice(0, 2);
+		const yotiSessionItem = new YotiSessionItem();
 		const yotiSessionId = randomUUID();
+
 		const lastYotiUuidChars = yotiSessionId.slice(-4);
-		this.logger.info("lastUuid", { lastUuidChars });
 		this.logger.info("lastYotiUuid", { lastYotiUuidChars });
+
+		//For IPV Integration happy path
+		if(IPV_INTEG_FULL_NAME_HAPPY === fullName){
+			//Replacing returned yoti sessionid with success 0100 at the end to return GBR_PASSPORT
+			yotiSessionItem.session_id = yotiSessionId.replace(lastYotiUuidChars, "0100");
+			return new Response(HttpCodesEnum.CREATED, JSON.stringify(yotiSessionItem));
+		}
+
+		//For all other cases
+		const lastUuidChars = fullName.match(/\d+/g)[0];
```

Comment:
this regexp doesn't return the last UUID characters ???

Reply:
modified var name